decentraland/unity-renderer
# fix: removed graphics auto-quality feature

## What does this PR change?

Removes the graphics auto-quality feature.

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/quality-preset-changes-automatically
2. Press tab or click on your avatar
3. Click on settings menu
4. You should not be able to see `Auto-Quality` toggle

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
